src/components/jump-to-banner/jump-to-banner.js

```diff
@@ -0,0 +1,29 @@
+const jumpToBanner = () => {
+	if (document.querySelector(".js-jump-to")) {
+		const jumpToBanners = document.querySelectorAll(".js-jump-to");
+
+		jumpToBanners.forEach((jumpToBanner)=> {
+			let jumpToLinks = jumpToBanner.querySelectorAll(".js-jump-to__link");
+	
+			jumpToLinks.forEach((jumpToLink) => {
+				jumpToLink.addEventListener("click", function (e) {
+					e.preventDefault();
+					handleLinkClick(jumpToLink);
+				});
+			});
+
+			const handleLinkClick = (jumpToLink) => {
```

Comment:
hm, i don't see any _need_ for this handler to be defined here?  if you could move it out of this function to the top level and just reference it, that would cut down on this function being 27 lines long.